cs3api4lab/handlers.py

```diff
@@ -60,7 +60,8 @@ def share_api(self):
     @web.authenticated
     @gen.coroutine
     def get(self):
-        yield RequestHandler.async_handle_request(self, self.share_api.list_shares, 200)
+        yield RequestHandler.async_handle_request(self, self.share_api.list_shares, 200,
+                                                  self.get_query_argument('filter', default='true') == 'true')
```

Comment:
filter powinna byc ustawiona na false,
filter nie jest dobra nazwa, jest zbyt ogolna i moze wrpowadzac w blad. To nie jest zestaw filtrow a konkretny parametr ktory powoduje jedna konkretna akcje, w tym miejscu lepsza nazwa bylo by "filter_duplicates"